software/ensemblvep/main.nf

```diff
@@ -0,0 +1,62 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options = initOptions(params.options)
+params.use_cache = false
```

Comment:
Why do these have to be `params` as opposed to explicit `input` options? If possible, it would be good to have the latter because that is how we have done things for all other modules to keep the option passing as simple as possible.

Reply:
I tried that at the beginning, and it wasn't working, but I could try it again now that I managed to make it work